### ISSUE TITLE
parsing enums with deprecated values fails with "syntax error"

### ISSUE THREAD

**Issue by paulswartz** [tomas-abrahamsson/gpb#179]
Parsing enums with (some kinds of) deprecated values fails with a syntax error:
```
enum e { 
  e1 = 1; 
  e2 = 2 [deprecated=true]; 
}

message m1 { 
  required e f1 = 1; 
}
```

I came across this issue trying to parse [gtfs-realtime.proto](https://github.com/google/transit/blob/master/gtfs-realtime/proto/gtfs-realtime.proto#L620):

```
gtfs-realtime.proto:620: syntax error before: deprecated
```

I can also reproduce it with a test case: https://github.com/paulswartz/gpb/commit/2ac2d5ff5b02

Adding spaces around the equal sign `[deprecated = true]` seems to work.

**Comment by tomas-abrahamsson** [comment]
Thanks providing a test case! Appreciated. I've pushed a fix to a branch, `fix-deprecated-enum-opt`, in case you'd want to try it out. For me, the `gtfs-realtime.proto` gets parsed successfullty on the branch. If you find it works well for you too, I'll include it in a new version.

**Comment by paulswartz** [comment]
Yep, that also parses for me. Thanks for the quick response!

**Comment by tomas-abrahamsson** [comment]
I included it in 4.11.2, which I just pushed, so I'm closing this ticket. Feel free to just reopen if there's anything more to do.
